lms/services/vitalsource/service.py

```diff
@@ -22,6 +26,8 @@ def __init__(
         user_lti_param: Optional[str] = None,
         user_lti_pattern: Optional[str] = None,
         enable_licence_check: bool = True,
+        enable_inclusive_access: bool = False,
+        inclusive_access_vbid: Optional[str] = None,
```

Comment:
I don't remember the details of VBID vs SKU but given this is about just one Hypothesis value it shouldn't be a problem to use the right value here.